src/main/java/io/confluent/connect/elasticsearch/ClientConfigCallbackHandler.java

```diff
@@ -210,4 +308,61 @@ private SSLContext sslContext() {
       }
     }
   }
+
+  private LoginContext login() throws PrivilegedActionException {
+    Configuration conf = new Configuration() {
+      @Override
+      public AppConfigurationEntry[] getAppConfigurationEntry(String name) {
+        return new AppConfigurationEntry[] {
+            new AppConfigurationEntry(
+                Krb5LoginModule.class.getName(),
+                AppConfigurationEntry.LoginModuleControlFlag.REQUIRED,
+                kerberosConfigs()
+            )
+        };
+      }
+    };
+
+    return AccessController.doPrivileged(
+        (PrivilegedExceptionAction<LoginContext>) () -> {
+          Subject subject = new Subject(
+              false,
+              Collections.singleton(new KerberosPrincipal(config.kerberosUserPrincipal())),
+              new HashSet<>(),
+              new HashSet<>()
+          );
+          LoginContext loginContext = new LoginContext(
+              "ElasticsearchSinkConnector",
+              subject,
+              null,
+              conf
+          );
+          loginContext.login();
+          return loginContext;
+        }
+    );
+  }
+
+  private Map<String, Object> kerberosConfigs() {
+    Map<String, Object> configs = new HashMap<>();
+    configs.put("useTicketCache", "false");
+    configs.put("useKeyTab", "true");
+    configs.put("keyTab", config.keytabPath());
+    //Krb5 in GSS API needs to be refreshed so it does not throw the error
+    //Specified version of key is not available
+    configs.put("refreshKrb5Config", "true");
+    configs.put("principal", config.kerberosUserPrincipal());
+    configs.put("storeKey", "false");
+    configs.put("doNotPrompt", "true");
+    configs.put("isInitiator", "true");
```

Comment:
So [here's my jaas config for Cassandra ](https://github.com/confluentinc/kafka-connect-cassandra/blob/master/src/main/java/io/confluent/connect/cassandra/CassandraSessionFactoryImpl.java#L49-L56).  
I set the following because I think the connector needs to refresh the TGT.  WDYT?
`   loginConfig.put("useTicketCache", "true");
      loginConfig.put("renewTGT", "true");`